org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/AdvancedNewProjectPage.xtend

```diff
@@ -187,9 +213,16 @@ class AdvancedNewProjectPage extends WizardPage {
 				validate(e)
 			}
 		})
+		createTestProject.addSelectionListener(new SelectionAdapter() {
+			override widgetSelected(SelectionEvent e) {
+				val enabled = createTestProject.selection==true
```

Comment:
==true seems to be redundant

Reply:
yes, I did this by intention because you could not read from the name that 'selection' is actually a boolean